controllers/vspherevm_controller_test.go

```diff
@@ -230,13 +229,13 @@ func TestReconcileNormal_WaitingForIPAddrAllocation(t *testing.T) {
 			Network:  nil,
 		}, nil)
 		r := setupReconciler(fakeVMSvc)
-		_, err = r.Reconcile(context.Background(), ctrl.Request{NamespacedName: util.ObjectKey(vsphereVM)})
+		_, err = r.Reconcile(ctx, ctrl.Request{NamespacedName: util.ObjectKey(vsphereVM)})
```

Comment:
Same question as in the other thread: use the test-package context from `ctx     = ctrl.SetupSignalHandler()` or should we create a context during the test instead (to match previous behaviour)

Reply:
discussing in another thread